packages/ag-charts-enterprise/src/features/zoom/zoomUtils.ts

```diff
@@ -181,9 +181,9 @@ export function constrainAxisWithOld(
     } else if (min === old.min) {
         max = min + minRatio;
     } else {
-        const cx = old.min + (old.max - old.min) / 2;
-        min = cx - minRatio / 2;
-        max = cx + minRatio / 2;
+        const c = min + (max - min) / 2;
+        min = c - minRatio / 2;
+        max = c + minRatio / 2;
```

Comment:
Previously, when the new zoom attempt was too far zoomed in, it would centre it on the old zoom, i.e. the centre of the chart. This fixes it to centre on the centre of the new zoom, but at the min ratios.